exercises/perfect-numbers/runtests.jl

```diff
@@ -0,0 +1,42 @@
+using Test
+include("example.jl")
+
+@testset "Smallest perfect number is classified correctly" begin
+   @test classify(6) == "perfect"
+end
+@testset "Medium perfect number is classified correctly" begin
+   @test classify(28) == "perfect"
+end
+@testset "Large perfect number is classified correctly" begin
+   @test classify(33550336) == "perfect"
+end
```

Comment:
The canonical data groups these together. This should also be reflected in the Julia tests by using (nested) testsets.